### PR TITLE
if answer has same rating in two or three answer it wont guess you wi…

### PR DESCRIPTION
**Pull request by rollychop**
Negetive Question Notifier added

### CODE REVIEW

File: app/src/main/java/ai/loko/hk/ui/MainActivity.java [hunk 1]
```diff
@@ -163,8 +164,12 @@ public void onClick(View v) {
 
     private void checkForUpdates() {
         new AppUpdater(this)
+                .setDisplay(Display.DIALOG)
                 .setUpdateFrom(UpdateFrom.JSON)
-                .setUpdateJSON("https://raw.githubusercontent.com/SubhamTyagi/loco-answers/master/update-changelog.json")
+                .setButtonDismiss(null)
+                .setButtonDoNotShowAgain(null)
+                .setUpdateJSON("https://raw.githubusercontent.com/rollychop" +
+                        "/loco-answers/master/update-changelog.json")
```
Comment (SubhamTyagi) [inline app/src/main/java/ai/loko/hk/ui/MainActivity.java]: Why??

File: app/src/main/java/ai/loko/hk/ui/MainActivity.java [hunk 2]
```diff
@@ -265,12 +270,14 @@ public void onBoomButtonClick(int index) {
...
-            public void onBoomButtonClick(int index) {
-                startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("https://github.com/SubhamTyagi/loco-answers/releases/")));
-            }
-        }));
+        rightBmb.addBuilder(new HamButton.Builder().normalImageRes(R.drawable.ic_directions_run_black_24dp).subNormalText("Click here to go to github release page").normalText("Update.")
+                .listener(new OnBMClickListener() {
+                    @Override
+                    public void onBoomButtonClick(int index) {
+                        startActivity(new Intent(Intent.ACTION_VIEW,
+                                Uri.parse("https://github.com/rollychop/loco-answers/releases/")));
```
Comment (SubhamTyagi) [inline app/src/main/java/ai/loko/hk/ui/MainActivity.java]: ??

File: README.md [hunk 3]
```diff
@@ -1,14 +1,13 @@
 
-[![licence](https://img.shields.io/badge/Licence-GPLv3-red.svg)](https://github.com/SubhamTyagi/loco-answers/blob/master/LICENSE) ![version](https://img.shields.io/badge/Version-2.6.1-green.svg)[ ![HitCount](http://hits.dwyl.io/SubhamTyagi/loco-answers.svg)](http://hits.dwyl.io/SubhamTyagi/loco-answers)  ![GitHub All Releases](https://img.shields.io/github/downloads/SubhamTyagi/loco-answers/total.svg?style=social)
+[![licence](https://img.shields.io/badge/Licence-GPLv3-red.svg)](https://github.com/rollychop/loco-answers/blob/master/LICENSE) ![version](https://img.shields.io/badge/Version-2.6.1-green.svg)[ ![HitCount](http://hits.dwyl.io/rollychop/loco-answers.svg)](http://hits.dwyl.io/rollychop/loco-answers)  ![GitHub All Releases](https://img.shields.io/github/downloads/rollychop/loco-answers/total.svg?style=social)
```
Comment (SubhamTyagi) [inline README.md]: Why this ??

File: README.md [hunk 4]
```diff
@@ -18,25 +17,17 @@
 * Trivia Games Supported by OCR and Accessibility 
 ## Download
 * Before downloading please give this app a :star: by pressing ![Star](/.github/assets/star.PNG) it means a lot to me it boost me to build more features and help you guys.
-* [Download latest Apk File](https://github.com/SubhamTyagi/loco-answers/releases)  ![GitHub All Releases](https://img.shields.io/github/downloads/SubhamTyagi/loco-answers/total.svg?style=social)
+* [Download latest Apk File](https://github.com/rollychop/loco-answers/releases)  ![GitHub All Releases](https://img.shields.io/github/downloads/SubhamTyagi/loco-answers/total.svg?style=social)
```
Comment (SubhamTyagi) [inline README.md]: Same here!

File: app/build.gradle [hunk 5]
```diff
@@ -30,19 +30,19 @@ apply plugin: 'com.android.application'
...
 
-        versionCode 34
-        versionName '2.6.1'
+        versionCode 37
+        versionName '2.7'
 
         multiDexEnabled true
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
-        versionNameSuffix '-Chipmunk'
+        versionNameSuffix '-AginstChina'
```
Comment (SubhamTyagi) [inline app/build.gradle]: Sentiments 😃

File: update-changelog.json [hunk 6]
```diff
@@ -1,10 +1,12 @@
 {
-  "latestVersion": "2.6.1",
-  "latestVersionCode": "34",
-  "url": "https://github.com/SubhamTyagi/loco-answers/releases",
+  "latestVersion": "2.7",
+  "latestVersionCode": "37",
+  "url": "https://github.com/rollychop/loco-answers/releases",
```
Comment (SubhamTyagi) [inline update-changelog.json]: Why?? May be your repo stuf

**Review by SubhamTyagi — CHANGES_REQUESTED** [review]

**Comment by SubhamTyagi** [comment]
Could you provide the apk file for this MR

**Comment by SubhamTyagi** [comment]
> 
> 
> Negetive Question Notifier added

What is this? Could you explain this more

**Comment by rollychop** [comment]
> > Negetive Question Notifier added
> 
> What is this? Could you explain this more

This will do nothing however notify if question is negative or if default search engine fails it will notify
if src/main/res/layout/floating.xml changes then user will get more from above 
this will enable highlighting of negative questions and many more

**Pull request closed without merging**